runtime/common/src/main/java/org/apache/nemo/runtime/common/metric/MetricUtils.java

```diff
@@ -84,7 +84,7 @@ private MetricUtils() {
    *
    * @return the loaded BiMaps, or initialized ones.
```

Comment:
Please update the description of the return parameter.

Reply:
Done 👍